azurerm/resource_arm_service_fabric_cluster.go

```diff
@@ -423,6 +433,7 @@ func resourceArmServiceFabricClusterRead(d *schema.ResourceData, meta interface{
 		d.Set("management_endpoint", props.ManagementEndpoint)
 		d.Set("reliability_level", string(props.ReliabilityLevel))
 		d.Set("upgrade_mode", string(props.UpgradeMode))
+		d.Set("cluster_code_version ", props.ClusterCodeVersion)
```

Comment:
I believe the test failure here is because there's a space at the end of this string (so the `d.Set` can't find a matching property in the schema called `cluster_code_version ` [note the space] - so the Set fails)?

Reply:
@steve-hawkins since this PR otherwise LGTM - I hope you don't mind but I've pushed a commit to remove the extra space (and will kick off the tests now)